osh/worker/csmock_runner.py

```diff
@@ -151,6 +166,15 @@ def srpm_download_analyze(self, analyzers, srpm_name, srpm_url, profile=None,
         srpm_path = self.download_file(srpm_url, srpm_name)
         return self.analyze(analyzers, srpm_path, profile, su_user, additional_arguments, **kwargs)
 
+    def dist_git_url_analyze(self, analyzers, dist_git_url, profile=None,
+                             su_user=None, additional_arguments=None, **kwargs):
+        if profile != "cspodman":
+            print("dist-git URL scan is incompatible with profile:", profile, file=sys.stderr)
+            return None, 2
+        # sanitize the URL before appending to the shell command
+        sanitized_url = shlex.quote(dist_git_url)
```

Comment:
Could you please squash the commit f0e56d5093a051e50cd55b6b85af326d786bd5d8, which introduces the sanitization?  Since this was discovered in the review, I do not want a commit that fixes a shell injection issue introduced by a previous commit in the final branch.

Reply:
done